src/Datadog.Trace/Tracer.cs

```diff
@@ -65,6 +65,13 @@ internal Tracer(TracerSettings settings, IAgentWriter agentWriter, ISampler samp
             AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
             AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
             Console.CancelKeyPress += Console_CancelKeyPress;
+
+            // If configured, add/remove the correlation identifiers into the
+            // LibLog logging context when a scope is activated/closed
+            if (Settings.LogsInjectionEnabled)
+            {
+                new LibLogCorrelationIdentifierScopeSubscriber(_scopeManager);
```

Comment:
How would you feel about a static method to express intent?

Reply:
I think that's a great idea. How would the method name `InjectLibLobScopeSubscriber` sound?

Reply:
That or something along the lines of `BootstrapLibLogScopeSubscriber` or `InitializeLibLogScopeSubscriber` or `SwtupLibLogScopeSubscriber`
Whatever you prefer.